PhysicsTools/PatAlgos/python/slimming/MicroEventContent_cff.py

```diff
@@ -77,27 +77,37 @@
         'keep CTPPSLocalTrackLites_ctppsLocalTrackLiteProducer_*_*'
     )
 )
-MicroEventContentMC = cms.PSet(
-    outputCommands = cms.untracked.vstring(MicroEventContent.outputCommands)
-)
-MicroEventContentMC.outputCommands += [
-        #'keep *_slimmedGenJets*_*_*',
-        'keep *_slimmedGenJets__*',
-        'keep *_slimmedGenJetsFlavourInfos_*_*',
-        'keep *_slimmedGenJetsAK8__*',
+
+MicroEventContentGEN = cms.PSet(
+    outputCommands = cms.untracked.vstring(
         'keep patPackedGenParticles_packedGenParticles_*_*',
         'keep recoGenParticles_prunedGenParticles_*_*',
         'keep LHEEventProduct_*_*_*',
-        'keep PileupSummaryInfos_slimmedAddPileupInfo_*_*',
         'keep GenFilterInfo_*_*_*',
         'keep GenLumiInfoHeader_generator_*_*',
         'keep GenLumiInfoProduct_*_*_*',
         'keep GenEventInfoProduct_generator_*_*',
+        'keep *_slimmedGenJets_*_*',
+        'keep *_slimmedGenJetsAK8_*_*',
+        'keep *_genMetTrue_*_*',
         # RUN
         'keep LHERunInfoProduct_*_*_*',
         'keep GenRunInfoProduct_*_*_*',
-        'keep L1GtTriggerMenuLite_l1GtTriggerMenuLite__*',
-]
+    )
+)
+
+MicroEventContentMC = cms.PSet(
+    outputCommands = cms.untracked.vstring(MicroEventContent.outputCommands)
+)
+MicroEventContentMC.outputCommands += MicroEventContentGEN.outputCommands
+MicroEventContentMC.outputCommands += [
+                                        'keep *_slimmedGenJets__*',
+                                        'keep *_slimmedGenJetsFlavourInfos_*_*',
```

Comment:
- ``` 'keep *_slimmedGenJets__*' ``` is redundant here, please remove
- [as mentioned above] ```'keep *_slimmedGenJetsFlavourInfos_*_*'``` perhaps should be added to MicroEventContentGEN